test/libsolidity/util/TestFunctionCall.cpp

```diff
@@ -225,6 +225,17 @@ string TestFunctionCall::format(
 		if (_hooks != nullptr)
 			for (auto& hook: *_hooks)
 				stream << hook->formatFunctionCall(*this, _errorReporter, _linePrefix, _renderResult, _highlight);
+
+		if (!m_reactions.empty())
+			stream << std::endl;
+
+		for (string const& reaction: m_reactions)
+		{
+			soltestAssert(reaction.find('.') == string::npos, "'.' not allowed in reaction.");
+			stream << _linePrefix << "// - " << reaction << ".";
```

Comment:
```suggestion
			stream << _linePrefix << " - " << reaction << ".";
```

Reply:
Are you sure that the reactions should be outside the isoltest specific comments?

Reply:
Ah sorry! Of course we need the slashes!